src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```

Comment:
Refer to the previous comment.

Reply:
I meant as 'does not belong in the project so consider removing the file' part, not the comment in code part haha, my mistake!